tests/std/tests/P0896R4_istream_view/test.cpp

```diff
@@ -73,16 +74,51 @@ void test_one_type() {
     ranges::copy(empty_constructed, input_empty);
     assert(ranges::equal(input_empty, expected_empty));
 
-    istringstream intstream{"0 1 2 3"};
-    T input_value[] = {-1, -1, -1, -1, -1};
-    ranges::copy(basic_istream_view<T, char>{intstream}, input_value);
-    assert(ranges::equal(input_value, expected));
+    { // using ranges::basic_istream_view with wide stream
+        wistringstream wintstream{L"0 1 2 3"};
+        T input_value[] = {-1, -1, -1, -1, -1};
+        ranges::copy(basic_istream_view<T, wchar_t>{wintstream}, input_value);
+        assert(ranges::equal(input_value, expected));
+    }
+
+    { // using ranges::basic_istream_view with narrow stream
+        istringstream intstream{"0 1 2 3"};
+        T input_value[] = {-1, -1, -1, -1, -1};
+        ranges::copy(basic_istream_view<T, char>{intstream}, input_value);
+        assert(ranges::equal(input_value, expected));
+    }

```

Comment:
Ultra nitpicks, no change requested: (1) I observe that these are wide-then-narrow while the following tests are ordered narrow-then-wide, and (2) I observe that these tests don't verify `static_assert(noexcept)` while the following tests do. (For the latter, there is no loss of coverage since `istream_view` and `wistream_view` are alias templates for `basic_istream_view`.)

This isn't significant enough to reset testing, just wanted to comment on *something*. :joy_cat:

Reply:
> Ultra nitpicks, no change requested: (1) I observe that these are wide-then-narrow while the following tests are ordered narrow-then-wide, and (2) I observe that these tests don't verify `static_assert(noexcept)` while the following tests do. (For the latter, there is no loss of coverage since `istream_view` and `wistream_view` are alias templates for `basic_istream_view`.)
> 
> This isn't significant enough to reset testing, just wanted to comment on _something_. 😹

I agree this just doesn't _quite_ rise to the level of resetting testing. I'll keep the comment open just in case someone requests another change in which case I'll address this as well.